### PR TITLE
reinstall-template 4.0 update

### PR DESCRIPTION
**Pull request by awokd**
add 4.0 clone template procedure because dummy template no longer works

### CODE REVIEW

**Comment by awokd** [comment]
Please pend this PR until https://github.com/QubesOS/qubes-issues/issues/3553, or let me know if I should close this one and open anew. I need to rewrite the reinstall portion. Thought it was R4.0 vs. R3.2 but it's actually Debian vs. Fedora updateVM.

**Comment by awokd** [comment]
Renamed sections so this should accurately describe how one would reinstall a template today. Can touch again later if #3553 changes anything, but I think this is good as is right now.

File: managing-os/reinstall-template.md [hunk 1]
```diff
@@ -9,72 +9,106 @@ redirect_from:
...
 =============================
 
-If you suspect your [TemplateVM] is broken, misconfigured, or compromised, you
-can reinstall any TemplateVM that was installed from the Qubes repository.
-Starting in Qubes 3.1, the process is greatly simplified.
+If you suspect your [TemplateVM] is broken, misconfigured, or compromised, you can reinstall any TemplateVM that was installed from the Qubes repository.
 
-First, copy any files that you wish to keep from the TemplateVM's `/home` and
-`/rw` folders to a safe storage location. Then, in a dom0 terminal, run:
+The procedure varies by Qubes version and UpdateVM's distribution; see the appropriate section below.
```
Comment (marmarek) [inline managing-os/reinstall-template.md]: IMO it would be good to point here how to check what is set as UpdateVM.

**Comment by awokd** [comment]
Added, but I'm just going to have to remove it again once --action=reinstall works on Debian!

**Pull request merged**